aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/NetworkUtilsSteps.java

```diff
@@ -77,6 +78,32 @@ public void setDeviceMqtt(final boolean connectivity) throws IOException, Interr
         }
     }
 
+    /**
+     * Add loop back address.
```

Comment:
```suggestion
     * Add IP address to loopback interface
```

Reply:
fixed in b0f3f75a79ffe49f7ffbb15e0ea553cc9deb6a7a